frontend/src/composables/useDomRef.ts

```
@@ -0,0 +1,11 @@
+import { computed, type Ref, shallowRef } from 'vue';
+import { type MaybeComputedElementRef, unrefElement } from '@vueuse/core';
+
+export function useDomRef<E extends HTMLElement | null>(): Ref<E, MaybeComputedElementRef | null> {
+  const domRef = shallowRef<E>(null!);
```

Biome (1.9.4)

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)